README.md

```diff
@@ -112,6 +128,7 @@ VueScrollTo.setDefaults({
 })
 ```
 
+### Browser and Vue 2.x or 3.x
```

Comment:
```suggestion
### Browser and Vue 2.x
```

It wouldn't work in the browser like that with 3.x - since there's no global Vue to register the directive on - they are local to each `createApp` call. Perhaps we could mention that and show example with:
```js
app.directive('scroll-to', VueScrollTo)
```
(I believe that should work - as long as you have access to the `app`)